portfolio/src/main/webapp/script.js

```diff
@@ -56,7 +56,17 @@ function openContact(){
 async function serverCommunication(){
   const serverResponse = await fetch("/data");
   console.log(serverResponse);
-  const responseToText = await serverResponse.text();
-  console.log(responseToText);
-  document.getElementById("greeting-container").innerHTML = responseToText;
+  const responseToJson = await serverResponse.json();
+  
+  //create object with response from server
+  myObject = responseToJson;
+
+  //access different elements using [] since it's an array
+  console.log(myObject[0]);
```

Comment:
For now, it may be fine to use an array with hard-coded size, especially when you just started.

Going forward, try to think how you could avoid this. Think about the situation where the server side may be updated to output more objects, then your client code needs to change. Not to mention if object size is dynamic.